test/puppetlabs/puppetdb/http/catalogs_test.clj

```diff
@@ -103,7 +106,7 @@
             expected (get paging-options p)]
         (is (= (map :certname expected) (map :certname response-body)))))))
 
-  (testing "/v4 endpoint is still responsive to old-style node queries"
-    (let [{:keys [body]} (get-response "/v4/catalogs" "myhost.localdomain")
+  (testing "endpoint is still responsive to old-style node queries"
+    (let [{:keys [body]} (get-response endpoint "myhost.localdomain")
```

Comment:
This is the response I get:
}[~/Documents/puppetdb] (ticket/master/pdb-1228) $ curl -X GET http://localhost:8080/v4/catalogs/mbp.local
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=ISO-8859-1"/>
<title>Error 500 </title>
</head>
<body>
<h2>HTTP ERROR: 500</h2>
<p>Problem accessing /v4/catalogs/mbp.local. Reason:
<pre>    Value does not match schema: {:edges {:data (not (sequential? a-clojure.lang.PersistentHashSet))}}</pre></p>
<hr /><i><small>Powered by Jetty://</small></i>
</body>
</html>